src/index.ts

```diff
@@ -36,5 +37,23 @@ rescue.from = function rescuefrom (constructor, callback) {
   }
 }
 
+rescue.all = function rescue (callbacks) {
+  return callbacks.map((callback) => {
+    return async function rescuehandler (...args: any[]): Promise<void> {
+      const next = args.slice(-1).pop() as NextFunction
+  
+      if (typeof next !== 'function') {
+        throw new TypeError('The last parameter received by express-rescue is not a function. Are you sure you passed its return as a middleware?')
+      }
+  
+      try {
+        await callback(...args) // eslint-disable-line
+      } catch (err) {
+        next(err)
+      }
+    }
+  })
+}
```

Comment:
This is unnecessary. In your PR's readme you demonstrated that `rescue.all` is equivalent to `[rescue(a), rescue(b), ...]` therefore we just need to do that:

```js
rescue.all = function rescueAll(callbacks) {
  return callbacks.map(rescue)
  // that's just short for:
  // return callbacks.map((middleware) => rescue(middleware))
}
```